highfidelity/hifi
# Cancel mode for teleporting

This PR makes it so you can cancel your teleport.  It also has nice new models for both the teleport and cancel overlays.

To test:
1. Press center thumb (vive) (or 1 on hydra ) while not moving to enter teleport mode.
2. Move the beam close to you.  It should be yellow and read cancel.  Releasing here should not teleport you.
3. Move the beam further.  It should read teleport and be blue.  Releasing here should teleport you.
